Src/PerformanceCollector/Shared/Implementation/QuickPulse/Process/IQuickPulseTopCpuCollector.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation.QuickPulse
+{
+    using System;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Interface for top CPU collector.
+    /// </summary>
+    internal interface IQuickPulseTopCpuCollector
+    {
+        bool InitializationFailed { get; }
+
+        bool AccessDenied { get; }
+
+        IEnumerable<Tuple<string, int>> GetTopProcessesByCpu(int topN);
```

Comment:
There's a school of thought (that I personally subscribe to) that less comments is better, so whenever you think object names don't make sense - leave a comment, that's what PRs are for :)